### PR TITLE
Changed $--outputDir to be required.

### PR DESCRIPTION
**Pull request by TaliVeith**

### CODE REVIEW

File: dark/graphics.py [hunk 1]
```diff
@@ -667,7 +667,10 @@ def alignmentPanelHTML(titlesAlignments, sortOn='maxScore',
         if not S_ISDIR(os.stat(outputDir).st_mode):
             raise ValueError("%r is not a directory." % outputDir)
     else:
-        os.mkdir(outputDir)
+        if outputDir == None:
```
Comment (terrycojones) [inline dark/graphics.py]: This should be `if outputDir is none`.

File: dark/graphics.py [hunk 2]
```diff
@@ -667,7 +667,10 @@ def alignmentPanelHTML(titlesAlignments, sortOn='maxScore',
...
+            raise NameError("The outputDir needs to be specified.")
```
Comment (terrycojones) [inline dark/graphics.py]: And you should raise a `ValueError` and document it in the docstring at the start of the function.

File: dark/graphics.py (hunk 1 above)
Comment (TaliVeith) [reply dark/graphics.py]: done

File: dark/graphics.py (hunk 2 above)
Comment (TaliVeith) [reply dark/graphics.py]: done

**Comment by terrycojones** [comment]
Looks good.

**Comment by TaliVeith** [comment]
Fixes #585

**Pull request merged**